spring-boot-tests/spring-boot-smoke-tests/spring-boot-smoke-test-tomcat-ssl/src/test/java/smoketest/tomcat/ssl/SampleTomcatSslApplicationTests.java

```diff
@@ -49,4 +49,24 @@ void testHome() {
 		assertThat(entity.getBody()).isEqualTo("Hello, world");
 	}
 
+	@Test
+	void testSslInfo() {
+		ResponseEntity<String> entity = this.restTemplate.getForEntity("/actuator/info", String.class);
+		assertThat(entity.getStatusCode()).isEqualTo(HttpStatus.OK);
+		assertThat(entity.getBody()).contains("\"name\":\"ssldemo\"")
+			.contains("\"alias\":\"spring-boot-ssl-sample\"")
+			.contains("\"status\":\"EXPIRED\"")
+			.contains("\"subject\":\"CN=localhost,OU=Unknown,O=Unknown,L=Unknown,ST=Unknown,C=Unknown\"");
```

Comment:
Is there a test tool in Boot that can do better verifications like this?
E.g.: Verifying values via JSONPath, RestAssured has a nice DSL around this.

Reply:
The paths aren't right in this example, but Framwork 6.2's `JsonContent` is an option here:

```java
JsonContent body = new JsonContent(entity.getBody());
assertThat(body).extractingPath("alias").isEqualTo("spring-boot-ssl-sample");
assertThat(body).extractingPath("status").isEqualTo("EXPIRED");
assertThat(body).extractingPath("subject")
	.isEqualTo("CN=localhost,OU=Unknown,O=Unknown,L=Unknown,ST=Unknown,C=Unknown");
```

Reply:
Oh, this is so much better, thank you!